packages/server/test.env

```diff
@@ -1,2 +1,2 @@
-LOG_ENABLED=true
+LOG_ENABLED=false
```

Comment:
Just want to note that I have been enjoying the ability to go through test log output, thanks for sneaking that in during #209!

Reply:
🕺 
for sure! glad it's been helpful. I haven't used it much but when ive needed it, it's been great